controller/runner.go

```diff
@@ -197,3 +201,13 @@ func (r *ServerRunner) Close() (err error) {
 
 	return nil
 }
+
+func outputFromConfig(config output.Config, queue *queue.Queue) Output {
+	if config.Console != nil && config.Console.Enabled {
+		return output.NewConsole(queue)
+	}
+	if config.Elasticsearch != nil {
```

Comment:
Is there a reason we're just checking for `!= nil` ? I'm imagining a scenario where someone sets `output.elasticsearch.enabled: false` and it still starts up or something.